CMakeLists.txt

```diff
@@ -152,6 +195,9 @@ find_package(Qt5 REQUIRED
     Gui
     Widgets
 )
+if(WIN32)
+set_source_files_properties("${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME}_autogen/mocs_compilation.cpp" PROPERTIES COMPILE_FLAGS "-include config.h -DGNULIB_NAMESPACE=gnulib")
```

Comment:
Normally SLiMgui is built with qmake, not cmake, which handles the MOC stuff and might handle other issues you're dealing with here.  But hey, if cmake works for you, no worries.